src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static final int MAX_NUMBER_OF_BALLS = 100;
+
+    public static Ball getRandomBall() {
+        return new Ball(ColorSupplier.getRandomColor(), new Random().nextInt(MAX_NUMBER_OF_BALLS));
```

Comment:
we should create an object of ColorSupplier and use it for calling the method getRandomColor().
Example:
```
ColorSupplier colorSupplier = new ColorSupplier();
...
colorSupplier.getRandomColor()
```